src/pages/apj-nerd-days.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { Link } from 'gatsby';
+import cx from 'classnames';
+import PageLayout from '../components/PageLayout';
+import MarketoForm from '../components/MarketoForm';
+import SEO from '../components/Seo';
+import nerdDays from '../images/nerd-days/nerd-days.png';
+import styles from './nerd-days.module.scss';
+
+const NerdDaysPage = () => {
+  return (
+    <>
+      <SEO />
+      <PageLayout type={PageLayout.TYPE.SINGLE_COLUMN}>
+        <PageLayout.Header title="New Relic Nerd Days APJ" />
+        <PageLayout.Content>
+          <section className={cx(styles.section, styles.twoColumnAlt)}>
+            <div>
+              <img
+                className={styles.img}
+                src={nerdDays}
+                alt="nerd days header"
+              />
+              <p>
+                Nerd Days is a <b>FREE</b> engineering conference that kicks off
+                October 22 <em>(Dates vary by region)</em>. Focused on building
```

Comment:
Since the nerd day pages seem to be identical except for varying the date and region, it might be worth it to pull this out into a template or component and then just render the template for each page in here with the props passed in.

Reply:
The content and structure will vary per page. They are just the same for now.